doc/changes/changes_2.1.0.md

```diff
@@ -1,9 +1,11 @@
-# Transformers Extension 2.1.0, t.b.d
+# Transformers Extension 2.1.0, 2024-10.22
```

Comment:
This is the most unusual date format. Personally I don't mind but I am not sure if the release droid will be happy with it.